### PR TITLE
docs: add --token permissions info

### PR DESCRIPTION
**Pull request by shroomist**

### CODE REVIEW

File: docs/org-stats.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Important notes:
 * The `--include-reviews` only grabs reviews from users that had contributions on the previous step.
 * In the `--blacklist` option, 'foo' blacklists both the 'foo' user and 'foo' repo, while 'user:foo' blacklists only the user and 'repo:foo' only the repository.
 * The `--since` option accepts all the regular time.Durations Go accepts, plus a few more: 1y (365d), 1mo (30d), 1w (7d) and 1d (24h).
-
```
Comment (caarlos0) [inline docs/org-stats.md:20]: I believe its only needed for private repos to work, public repos should work regardless... maybe we should make that explicit here too

File: docs/org-stats.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Important notes:
...
+* The `--token` token permissions need to include 'repo - Full control of private repositories'.
```
Comment (shroomist) [inline docs/org-stats.md:20]:
```suggestion
* The `--token` token permissions need to include 'repo - Full control of private repositories'. Required if you need to fetch data from private repositories in your organisation
```

File: docs/org-stats.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ Important notes:
 * The `--include-reviews` only grabs reviews from users that had contributions on the previous step.
 * In the `--blacklist` option, 'foo' blacklists both the 'foo' user and 'foo' repo, while 'user:foo' blacklists only the user and 'repo:foo' only the repository.
 * The `--since` option accepts all the regular time.Durations Go accepts, plus a few more: 1y (365d), 1mo (30d), 1w (7d) and 1d (24h).
-
+* The `--token` token permissions need to include 'repo - Full control of private repositories'. Required only if you need to fetch data from private repositories in your organisation
```
Comment (shroomist) [inline docs/org-stats.md:20]:
```suggestion
* The `--token` token permissions need to include 'repo - Full control of private repositories'. Required only if you need to fetch data from private repositories in your organisation.
```

**Comment by caarlos0** [comment]
thanks!

**Pull request merged**